### DISCUSSION TITLE
Scale factor / other UI thoughts

### DISCUSSION THREAD

**Discussion by Kalekki** [zero01101/openOutpaint#29] [Ideas]
I think it would make more sense to instead of showing scale factor in the UI, just show the resulting image size.
Also capturing scroll events while in mask mode for the purpose of changing the brush size to be independent of scale factor

**Comment by Kalekki** [comment]
~~While on the topic of UI, having a checkbox for overmasking could(should?) be made redundant with just checking if the value > 0~~ fixed

**Comment by Kalekki** [comment]
~~Instead of having the mask brush be translucent, have the canvas be certain transparency, so the mask drawing doesnt have variable transparency depending how slow/many times you draw on a spot~~ [fixed](https://github.com/zero01101/openOutpaint/pull/22)

**Comment by zero01101** [comment]
completely agreed on the control being kind of a crappy workaround, mousewheel and cursor size would definitely be nicer; actually just pushed [a commit](https://github.com/zero01101/openOutpaint/commit/cf07ef208c6693d0b98a05515241078c9a1f2735) removing the overmask toggle outright because i had kind of the same thought last night :) the mask brush blob is 100% currently "minimum viable product" lol and will likely experience a significant quantity of life-altering events :D
